core/shared/src/main/scala/io/circe/Decoder.scala

```diff
@@ -1,11 +1,13 @@
 package io.circe
 
+import java.nio.ByteBuffer
```

Comment:
Just merge this with the rest of the imports. I'm not sure Circe's scalastyle settings allow this.